lib/lighthouse/benefits_education/outside_working_hours.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'common/exceptions/base_error'
+
+module BenefitsEducation
+  ##
+  # Custom error for when the user is attempting to access the service
+  # outside of working hours.  The service proxies to a service which
+  # has nightly downtime
+  class OutsideWorkingHours < Common::Exceptions::BaseError
+    ##
+    # @return [Array[Common::Exceptions::SerializableError]] An array containing the error
+    #
+    def errors
+      [Common::Exceptions::SerializableError.new(i18n_data)]
+    end
+
+    ##
+    # @return [Time] The time to retry the request
+    #
+    def retry_after
+      Time.now.httpdate.in_time_zone('Eastern Time (US & Canada)').to_s
```

Comment:
Why is this returning the current time?

Reply:
Hey Steve! Good catch, that code's not even used! I copied it from elsewhere 🙄 

The *actual* code to calculate/notify user when they should retry is [here](https://github.com/department-of-veterans-affairs/vets-api/blob/2a4f626813c4978f55767773f53ef9bf6c17c86e/lib/lighthouse/benefits_education/service.rb#L94-L106).  

That code will be utilized to add `Retry-After` headers (coming in the next PR!)